bigtable/integration_test.go

```diff
@@ -2190,3 +2192,23 @@ func clearTimestamps(r Row) {
 		}
 	}
 }
+
+func deleteTable(ctx context.Context, t *testing.T, ac *AdminClient, name string) {
+	bo := gax.Backoff{
+		Initial:    100 * time.Millisecond,
+		Max:        2 * time.Second,
+		Multiplier: 1.2,
+	}
+	ctx, _ = context.WithTimeout(ctx, time.Second*5)
```

Comment:
I'm concerned that 5s may be too short of a timeout still-- in some cases this might cause a single call to timeout before it completes if latency is high. Let's choose a higher value (say 30s) and see how that does.

Reply:
No problem, pushed